hknweb/candidate/views.py

```diff
@@ -48,6 +48,7 @@ def get_context_data(self):
         num_rejected = challenges \
                 .filter(Q(officer_confirmed=False) | Q(csec_confirmed=False)) \
                 .count()
+        num_confirmed = challenges.count() - num_pending - num_rejected
```

Comment:
Does this work? I've gotten errors about QuerySets being evaluated twice like this.

Reply:
Uh hasn't errorred yet